### PR TITLE
solution making change

### PR DESCRIPTION
**Pull request by jimmyhu997**

### CODE REVIEW

**Review by storrisi — CHANGES_REQUESTED** [review]
Actually the requirement is not just a simple conversion from cents to other values, but to explain the way you can change an amount (for example , 5 euro) with the less amount of other values.

File: projects/m1/003-making-change/solutions/jimmyhu/making-change.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+payment_float = "{:.2f}".format(payment)
+cent= int(payment_float[-2:])
+print("the change could be: " + str(cent / pennie) + " pennies or " + str(cent / nickel) + " nickels or " + str(cent / dime) + " dimes or " + str(cent / quarter) + " quarters" )
+
+
+
+
+
+# import sys
+# def change_coins(coins, m, difference):
```
Comment (storrisi) [inline projects/m1/003-making-change/solutions/jimmyhu/making-change.py:17]: where this solution comes from?

**Comment by jimmyhu997** [comment]
I'm not sure if i have done right the push for the corrections

**Review by storrisi — APPROVED** [review]
It makes more sense now :)

**Pull request merged**